components/StoreList/index.js

```diff
@@ -0,0 +1,22 @@
+import StoreListItem from "../StoreListItem";
```

Comment:
Decide how to import but do not mix both variants.
![Screenshot 2023-11-24 at 12 38 06](https://github.com/GoekhanAgkurt/capstone-grocery-app/assets/111574974/a67f1bbd-21f9-4040-aaac-f5f543983e02)

Reply:
Yes, we decided to use the @ variant.